### PR TITLE
Add cite commands and overhaul bibliography

### PR DESCRIPTION
**Pull request by agahkarakuzu**
https://github.com/neurolibre/neurolibre-reviews/issues/17

### CODE REVIEW

**Comment by agahkarakuzu** [comment]
Also, think about which featured image to display on the card: 
image

You'll need to add it as `featured.png` at the root of the repo, then it'll show up.

File: content/index.ipynb [hunk 1]
```diff
@@ -56,43 +56,43 @@
...
+    "Various normalization strategies have been proposed to account for the above-mentioned factors on spinal cord morphometric measures. Sex was used for CSA normalization in several works {cite:p}`Bedard2022-tr,Kesenheimer2021-vw,Papinutto2020-hn,Rashid2006-ar`. Other studies proposed spinal cord length as a normalization factor {cite:p}`El_Mendili2023-ws,Martin2017-cf,Martin2017-zu,Rocca2019-yc`. Additionally, combining spinal cord length with a z-score normalization was proposed to account for variations along the superior-inferior axis {cite:p}`Martin2017-cf,Martin2017-zu`. Another approach taking into account the dependency of spinal cord anatomy on a level involved the normalization of morphometric measures from the compression site using non-compressed levels above and below {cite:p}`Guo2022-qb,Miyanji2007-za`. Finally, several studies normalized CSA using the spinal canal and brain metrics, including spinal canal area, spinal canal diameter, brain volume, intracranial volume, thalamic volume, and head size normalization factor {cite:p}`Bedard2022-tr,Horsfield2010-gx,Kesenheimer2021-vw,Papinutto2020-hn,Rashid2006-ar,Rocca2019-yc`. \n",
```
Comment (valosekj) [inline content/index.ipynb]:
```
Other studies proposed spinal cord length as a normalization factor {cite:p}`El_Mendili2023-ws,Martin2017-cf,Martin2017-zu,Rocca2019-yc`.
```
`Oh et al., 2014` is missing here

**Comment by valosekj** [comment]
Fantastic work, @agahkarakuzu! Thank you!

**Comment by agahkarakuzu** [comment]
@valosekj added the missing citation, also a `featured.png` of a figure I think looks really cool. If all looks good to you, feel free to merge.

**Comment by valosekj** [comment]
> @valosekj added the missing citation, also a `featured.png` of a figure I think looks really cool. If all looks good to you, feel free to merge.

Thank you! I replaced the `featured.png` by [fig1](https://github.com/valosekj/PAM50-normalized-metrics-paper/blob/main/content/images/fig1.png). --> merging

**Pull request merged**